Makefile

```diff
@@ -96,21 +96,21 @@ kind-cluster: ## Standup a kind cluster for e2e testing usage
 
 ##@ install/run:
 
-install-apis: cert-mgr generate kustomize ## Install the core rukpak CRDs
-	$(KUSTOMIZE) build manifests/apis | kubectl apply -f -
+install-apis: generate ## Install the core rukpak CRDs
+	$(KUSTOMIZE) build manifests/apis/crds | kubectl apply -f -
 
-install-plain: install-apis ## Install the rukpak CRDs and the plain provisioner
+install-plain: ## Install the rukpak CRDs and the plain provisioner
 	$(KUSTOMIZE) build manifests/provisioners/plain | kubectl apply -f -
 
-install-crdvalidator: cert-mgr kustomize ## Install the crdvalidator webhook manifests to the current cluster.
+install-crdvalidator-webhook: ## Install the crdvalidator webhook manifests to the current cluster.
 	$(KUSTOMIZE) build manifests/crdvalidator | kubectl apply -f -
 
-install: install-plain ## Install all rukpak core CRDs and provisioners
+install-core-webhook:
+	$(KUSTOMIZE) build manifests/apis/webhooks | kubectl apply -f -
 
-deploy: install-apis install-crdvalidator ## Deploy the operator to the current cluster
```

Comment:
Looking at the overall diff, it looks like we're removing the deploy Makefile target, but there's still a .PHONY deploy target?